### ISSUE TITLE
mutation Payloads

### ISSUE THREAD

**Issue by JakeIwen** [carlpeaslee/bouncetribe#22]
This seems to be common across many mutations. Why do I only get back a user ID... and their portrait? Where is this being set? Why their portrait?

Adding more fields to the mutation  fatQuery (that are seemingly acceptable based on the relay image below) doesn't actually yield more information in the response.  Sometimes it just gives an error (if for example i try to return 'viewer' instead of friends1user).
Can you shed any light on what dictates what fatQuery can return?

fatQuery results are  just data that has been modified right?

The response to a successful AddToFriends mutation looks like this:
![image](https://user-images.githubusercontent.com/20415982/32462711-f98575aa-c2ff-11e7-8eea-23c04b276201.png)

the mutation class:
![image](https://user-images.githubusercontent.com/20415982/32463010-1746fcd4-c301-11e7-8c07-487df901e927.png)

the Relay payload:
![image](https://user-images.githubusercontent.com/20415982/32462784-3bb3b78e-c300-11e7-9386-27baa5b3a15a.png)

@carlpeaslee

**Comment by carlpeaslee** [comment]
Yeah my understanding is that the FatQuery will generally only return data that has changed. I️ think one exception to this is if you set the getConfigs to ... I️ don’t remember what it is right now but it’s a relay confit setting that will force relay to give you back whatever data you specify in your fat query even if it didn’t change.
